app/Notifications/ChangeRequestVoidedNotification.php

```
@@ -0,0 +1,71 @@
+        ];
+    }
+
+    public function getToken()
+    {
+        return $this->token;
+    }
+
+    /**
+     * Get the mail representation of the notification.
+     */
+    public function toMail(object $notifiable): MailMessage
```

PHPMD (2.15.0)

Avoid unused parameters such as '$notifiable'. (undefined)

(UnusedFormalParameter)